docs/DeveloperGuide.md

```diff
@@ -2,36 +2,118 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}  
+Zhou Zhou's Individual Project: https://github.com/maanyos/ip \
+Nay Chi's Individual Project: https://github.com/NayChi-7/ip
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Class Diagrams
```

Comment:
For your class diagrams, you have the circle and A or C depending whether it is a class or an abstract class. This is actually not standard notation I think? You might want to hide the circle icons to fit into this module's standard notation. https://plantuml.com/class-diagram#Hide

Reply:
+1 to the comment above. You should hide the "A" / "C" circles.